src/app/pages/MainArticlesScreen/components/ArticleComponent/Article.component.tsx

```diff
@@ -15,19 +12,13 @@ import {
   ArticleMetaDataWrapper,
   ArticleMetaTitle,
 } from './Article.styles';
+import { OwnProps } from './index';
```

Comment:
OwnProps are needd only here so there is no need to import them from level up wrapper which is probably not using them at all.